data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1611 @@
+			choice
+				`	"Sure, I could do that."`
+					goto rc1accept
+				`	"Not right now, but I could later."`
+					defer
+				`	"How much will I get paid?"`
+			`	"Once the research is published, you will receive one million credits.`
+			choice
+				`	"I accept."`
+				`	"Not right now, but I could later."`
+					defer
+			label rc1accept
+			`	"Excellent. As before, please visit both Nasqueron and Slylandro, record the void sprites there, and return here."`
+				accept
+	npc
+		government "Drak"
+		system "Nenia"
+		personality staying heroic disables uninterested
+		ship "Archon (Cloaked)" "Lifted Lorax"
+	npc
+		government "Korath"
+		system "Nenia"
+		personality entering forbearing timid uninterested surveillance staying
+		fleet
+			names "korath"
+			cargo 1
+			commodities "void sprite parts"
+			variant
+				"Korath Raider (Ember)"
+				"Korath Chaser" 2
+	on enter "Nenia"
+		dialog
+			`As you scan the system, the first thing you notice is that the void sprites' behavior seems different: There are fewer of them flying around, and the few that you see seem to be almost timid - floating quietly, and dropping down into the cloud deck whenever your ship moves towards them. You recall that Plume asked you to visit both planets.`
+	on stopover
+		conversation
+			`	The void sprites here prove to be elusive. You spend quite a while chasing radar signatures before you finally manage to get a small flock of them into visual range. You record them from a distance for several minutes, until they suddenly disperse into a storm and vanish.`
+	on complete
+		conversation
+			`Disappointed in the void sprites' behavior, you deliver the recordings to Plume's lab without fanfare. Plume quickly takes and reviews the footage before making a gesture that you think means 'curious dejection.' Then he seems to notice something, and resumes his usual intent bearing.`
+			`	"Here, look!" He rewinds the recording and gestures curtly at the projection in front of you. "There. That one is injured. Some kind of burn in a long, straight line." He zips through the footage farther: "And here, a scattering of scorch marks." He looks up at you and snaps out a series of gestures: "This is outside my expertise; I will need to consult an expert. Meet me in the spaceport in an hour."`
+
+
+
+mission "Remnant: Cognizance 2"
+	name "Eliminate Korath Ship"
+	description "Plume ordered you to eliminate the Korath ship in the <waypoints> system."
+	source "Aventine"
+	waypoint
+		system "Nenia"
+	to offer
+		has "Remnant: Cognizance 1: done"
+	on offer
+		conversation
+			`You arrive in the spaceport to find Plume pacing quickly along a walkway. He gestures a quick greeting and jumps straight to the point. "We have never seen marks like these naturally appear on void sprites, but they match the weapon scoring patterns we have seen delivered by Korath ships. As such, we believe the Korath have begun hunting void sprites."`
```

Comment:
Scaring isn't the word I'm going for here.

Scoring, as in damaging, marking. From Mirriam-Webster:
> to mark with or as if with a line or groove

    the glassblower scored the glass rod first so that it would break cleanly